examples/vercel-ai-chatbot/app/api/chat/route.ts

```diff
@@ -151,7 +151,11 @@ export async function POST(req: Request) {
                   role: 'ASSISTANT' as const,
                   content: {
                     type: 'FUNCTION_CALL',
-                    value: functionCallItem.value
```

Comment:
long term, we ideally fix the API such that the response and request shapes are consistent for these values